fsspec/tests/test_compression.py

```diff
@@ -48,6 +48,11 @@ def test_infer_custom_compression():
         del compressions["tst"]
 
 
+def test_lzma_compression_name():
+    pytest.importorskip("lzma")
+    assert infer_compression("fn.xz") == "xz"
```

Comment:
OK, I suppose I was confused. The file extension is still xz, and the compression should be called xz, even though the module/codec is lzma? Should we perhaps include "lzma" in the list of compressions also?

Reply:
> The file extension is still xz, and the compression should be called xz, even though the module/codec is lzma?

That's my understanding based on the first paragraph of https://docs.python.org/3/library/lzma.html (although there's also a legacy `.lzma` extension which is also supported : / ). Naming the compressions "xz" is primarily to maintain compatibility with previous behavior. Adding an "lzma" compression seems reasonable too.

Reply:
Alright so should add it